datumaro/components/annotation.py

```diff
@@ -467,16 +468,16 @@ def extract(self, instance_id: int) -> IndexMaskImage:
     def lazy_extract(self, instance_id: int) -> Callable[[], IndexMaskImage]:
         return lambda: self.extract(instance_id)
 
-@attrs(order=False)
+@attrs(slots=True, order=False)
 class _Shape(Annotation):
     # Flattened list of point coordinates
-    points: List[float] = attrib(converter=lambda x:
-        [round(p, COORDINATE_ROUNDING_DIGITS) for p in x])
+    points: List[float] = field(converter=lambda x: \
+        np.round(x, COORDINATE_ROUNDING_DIGITS).tolist())
```

Comment:
FWIW, the canonical name for this function seems to be [`around`](https://numpy.org/doc/stable/reference/generated/numpy.around.html) / [`round_`](https://numpy.org/doc/stable/reference/generated/numpy.round_.html).

Reply:
Probably, it was renamed once, because `around` docs use `np.round` in examples

Reply:
Huh, weird. I guess NumPy developers can't resist the temptation to use these undocumented aliases either. :-) Maybe I'll file bug report for the NumPy docs.

Reply:
I found one that already exists: numpy/numpy#19717.